python/src/schemas/schema_V2024_05_24.py

```diff
@@ -699,7 +699,7 @@ class SubrecipientRow(BaseModel):
     Name: constr(strip_whitespace=True, min_length=1, max_length=80) = Field(
         ...,
         serialization_alias="Subrecipient Name",
-        json_schema_extra={"column":"C"}
+        json_schema_extra={"column":"C", "output_column": "B"}
     )
     Recipient_Profile_ID__c: Optional[constr(strip_whitespace=True, min_length=1, max_length=100)] = Field(
```

Comment:
Just calling out that `Recipient_Profile_ID__c` wasn't in the template, so I didn't add a column mapping here

Reply:
hmm I see this as column `C` https://docs.google.com/spreadsheets/d/1-CN8fyKCQQRf1QGcS-wXwCNVGIyhhLq2/edit#gid=1101132838

Reply:
Discussed offline and updated PR to reflect new template!